Frameworks/GLKit/GLKTexture.mm

```diff
@@ -277,7 +279,8 @@ @implementation GLKTextureLoader {
 */
 + (GLKTextureInfo*)textureWithContentsOfFile:(NSString*)fname options:(NSDictionary*)opts error:(NSError**)err {
     CGDataProviderRef provider = CGDataProviderCreateWithFilename([fname UTF8String]);
-    CGImageRef img = CGImageCreateWithPNGDataProvider(provider, NULL, NO, kCGRenderingIntentDefault);
+    unsigned char* dataBytes = static_cast<unsigned char*>(const_cast<void*>(_CGDataProviderGetData(provider)));
```

Comment:
Not super related to this, but _CGImageGetImageFromData returns a +1 rn which breaks the CF naming convention, so that needs to be changed at some point